modules/src/ics03_connection/msgs/conn_open_try.rs

```diff
@@ -30,8 +29,8 @@ pub struct MsgConnectionOpenTry {
     pub counterparty: Counterparty,
     pub counterparty_versions: Vec<Version>,
     pub proofs: Proofs,
-    pub delay_period: u64,
-    pub signer: AccountId,
+    pub delay_period: u64, // FIXME(romac): Introduce newtype for `delay_period`
```

Comment:
Yus! Probably the todo should be fixed alongside https://github.com/informalsystems/ibc-rs/issues/640.